src/IO.luna

```diff
@@ -71,6 +71,15 @@ class CSVParser:
         CustomNames l: l
         _: []
 
+    # Read CSV file from the hard drive, returning a `Table` value.
+    # 
+    # > import Dataframes.IO 
+    # > CSVParser.readFile "./data/simple_empty.csv"
```

Comment:
There should be information that by default:
1) first row is assumed to contain column names
2) column types are deduced